api/envoy/config/filter/network/direct_response/v2/config.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+package envoy.config.filter.network.direct_response.v2;
+
+import "udpa/annotations/migrate.proto";
+
+option java_package = "io.envoyproxy.envoy.config.filter.network.direct_response.v2";
+option java_outer_classname = "ConfigProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_migrate).move_to_package =
+    "envoy.extensions.filters.network.direct_response.v3";
+
+// [#protodoc-title: Direct response]
+// Direct response :ref:`configuration overview <config_network_filters_direct_response>`.
+// [#extension: envoy.filters.network.direct_response]
+
+message Config {
+  // Response data as bytes.
+  bytes response = 1;
```

Comment:
From a usability perspective I would probably do a oneof here and either allow a string or bytes, but up to you. (It would actually be kind of nice to have a StringOrBytes type that we use since we have this pattern/problem in other places.).

Reply:
I followed "not for direct human authoring" mantra here. I agree, it is a general problem with JSON/proto encoding.

Reply:
FWIW there's the `DataSource` type that's basically oneof inline bytes / inline string / filename.

Reply:
Hmm yeah. This would have been better. @kyessenov do you want to make the change? Given this was just added I think it's OK to switch it.

Reply:
Already done :) I wasn't aware of that resource. It's probably a good idea to promote it to UDPA.